### ISSUE TITLE
2.28-RELEASE not in Maven Central Repository

### ISSUE THREAD

**Issue by pmwmedia** [eclipse-ee4j/jersey#4008]
I cannot find 2.28-RELEASE in Maven Central Repository. Is there any plan to publish new releases there?

**Comment by jansupol** [comment]
Yes, the release is in maven central along with other Jakarta EE components, in January

**Comment by jansupol** [comment]
As you can see, some projects started to be released in central (JAX-B, JSON-B) so it's close. Since Jersey depends on Yasson, EclipseLink (Moxy), ... we need to be released when they are. My personal estimation is 2 weeks.

**Comment by hc-codersatlas** [comment]
2.28 is out now!

**Comment by usulkies** [comment]
Time to update the index page...
https://jersey.github.io/index.html

**Comment by jansupol** [comment]
Yes, we will update it, hopefully until the end of the week. As the API of Jersey has not been changed, everything on the webpage should work
